waiter/integration/waiter/token_request_test.clj

```diff
@@ -140,46 +164,68 @@
           (let [{:keys [body] :as response} (get-token router-url token :cookies cookies)]
             (assert-response-status response 404)
             (is (str/includes? (str body) "couldn't find token") (str body)))
+          (let [response (get-token router-url token
+                                    :cookies cookies
+                                    :query-params {"include-deleted" true})]
+            (assert-get-token-response-200 response service-id-prefix true))
           (let [{:keys [body] :as tokens-response} (list-tokens router-url current-user :cookies cookies)
                 tokens (json/read-str body)]
             (assert-response-status tokens-response 200)
-            (is (not-any? (fn [token-entry] (= token (get token-entry "token"))) tokens))))))))
+            (is (not-any? (fn [token-entry] (= token (get token-entry "token"))) tokens)))))
+
+      (log/info "deleting the tokens in excise mode")
```

Comment:
We keep on using the words "hard delete" to actually describe what we're doing (twice in this thread). There was also an occurrence of "erase" in a log statement somewhere else. 

Why not just use "hard delete"?